dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -964,22 +964,54 @@ public BaseObjectRest getBaseObjectRestFromUri(Context context, String uri) thro
             throw new IllegalArgumentException("the supplied uri lacks required path elements: " + uri);
         }
 
+        return findBaseObjectRest(context, uriParts[0], uriParts[1], uriParts[2]);
+    }
+
+    /**
+     * Gets the rest object of of specified type (i.e. "core.item", "core.collection", "workflow.workflowitem",...)
+     * having given uuid.
+     *
+     * @param context the DSpace context
+     * @param type Object type
+     * @param uuid Object uuid
+     * @return the {@link BaseObjectRest} identified by the provided uuid
+     */
+
```

Comment:
Tiny thing, there's an unnecessary empty line between the JavaDocs and the method